assets/src/scss/pages/_page.scss

```diff
@@ -121,3 +121,79 @@
     outline: 0 none;
   }
 }
+
+.bottom-navigation {
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-family: var(--font-family-primary);
+  font-size: var(--font-size-xxs--font-family-primary);
+
+  a {
+    display: flex;
+    align-items: center;
+  }
+
+  a:hover {
+    text-decoration: none;
```

Comment:
@Osong-Michael what about [this comment](https://github.com/greenpeace/planet4-master-theme/pull/2432/files#r1865601305)?

Reply:
@GP-Dan-Tovbein, Houssam asked to remove the underline from the block. To leave it as per the designs